server/db.mjs

```diff
@@ -0,0 +1,12 @@
+import dotenv from "dotenv";
+import pg from "pg";
+
+dotenv.config({ path: "../.env" });
```

Comment:
This is how we load the variables from `.env`. They get added to the Node built-in [`process.env`](https://nodejs.org/dist/latest-v14.x/docs/api/process.html#process_process_env) object